src/lane.cpp

```diff
@@ -93,28 +93,24 @@ void Lane::add_car(carType type)
 				new_car.set_rotation(PI / 2);
 				new_car.set_original_rotation(PI / 2);
 				new_car.set_lane(direction::NORTH);
-				//new_car.set_desired_direction(direction::EAST);
 			}
 			else if (m_dir == direction::WEST) {
 				new_car.set_position({ -130.f,537.f });
 				new_car.set_lane(direction::WEST);
 				new_car.set_rotation(0);
 				new_car.set_original_rotation(0);
-				new_car.set_desired_direction(direction::EAST);

```

Comment:
@yhymason Mason, you added this line (and 110 and 117). As the docs for set_desired_direction indicate, the function:

`// Sets the desired direction for the car and removes it's villainy`

Which means that none of the cars in those lanes were villains any more. set_desired_direction is what the AI calls to have the villains make their decisions. Also, adding these function calls would ostensibly cause all of the cars to go in only one direction, if it weren't for the generate_desired_direction() call on line 119.

Can you confirm that these were extraneously added and are fine to remove?

Reply:
Safe to remove, it's strange, I don't have these lines on my local files.